crates/api_models/src/payment_methods.rs

```diff
@@ -437,6 +437,8 @@ pub struct PaymentMethodListResponse {
         }
     ]
     ))]
+    pub merchant_name: OptionalEncryptableName,
```

Comment:
The schema information and documentation comments annotated to this field are that of the `payment_methods` field. Could you please fix that, and add schema information and doc comments for the `merchant_name` field?